stories/plotly/plotly-bar.stories.js

```diff
@@ -1,26 +1,28 @@
 import React from 'react'
 import Bar from '../../src/components/plotly/bar'
-import { mockData } from '../data/others/mock-data'
-import ParentSize from '@visx/responsive/lib/components/ParentSize'
+
+import mockData from '../data/plotly/mock-data-pie-bar'
+import ResponsiveChartWrapper from '../nivo/responsive-chart-wrapper'
```

Comment:
If `ResponsiveChartWrapper` is used by nivo and plotly charts, maybe move it out directly into stories folder.